packages/recorder-loader/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const SNIPPET_BASE_URL = "https://snippet.meticulous.ai";
```

Comment:
tiny nit - I believe we use plural elsewhere in the codebase.
```suggestion
export const SNIPPETS_BASE_URL = "https://snippet.meticulous.ai";
```